### DISCUSSION TITLE
Bulidit 맵 로더 이슈 - 500 Internal Server Error

### DISCUSSION THREAD

**Discussion by xowjd913** [naverz/zepeto-studio-kor#145] [Issue Report (Legacy)]
빌드잇 맵 로드 시 500 Error 후 로더 창이 나오지 않습니다.

![image](https://user-images.githubusercontent.com/25768804/147439563-7363be2c-39f0-4aeb-8eeb-0b0b35c5046c.png)

**Comment by hyeonjin-jo** [comment]
안녕하세요.

일시적인 서버 오류로 인해서 발생한 문제입니다.

이후 반복되지 않도록 체크 진행하겠습니다

감사합니다.
